Cue/app/tabs/library/LibraryListView.js

```diff
@@ -129,16 +125,16 @@ class LibraryListView extends React.Component {
         dataSource={this.state.dataSource}
         initialListSize={8}
         pageSize={2}
+        refreshControl={
+          <RefreshControl
+            refreshing={this.props.refreshing}
```

Comment:
Nitpick: Can we add `colors={[CueColors.primaryTint]}` to the `RefreshControl` to tint the inner loading circle on Android?